src/Symfony/Component/Messenger/Tests/Transport/Doctrine/DoctrineIntegrationTest.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Tests\Transport\Doctrine;
+
+use Doctrine\DBAL\DriverManager;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Messenger\Tests\Fixtures\DummyMessage;
+use Symfony\Component\Messenger\Transport\Doctrine\Connection;
+
+/**
+ * @requires pdo_mysql
+ */
+class DoctrineIntegrationTest extends TestCase
+{
+    private $driverConnection;
+    private $connection;
+
+    protected function setUp()
+    {
+        parent::setUp();
+
+        if (!getenv('MESSENGER_DOCTRINE_DSN')) {
+            $this->markTestSkipped('The "MESSENGER_DOCTRINE_DSN" environment variable is required.');
+        }
+        $dsn = getenv('MESSENGER_DOCTRINE_DSN');
+        $this->driverConnection = DriverManager::getConnection(['url' => $dsn]);
+        $this->connection = new Connection([], $this->driverConnection);
+        // call send to auto-setup the table
+        $this->connection->send('...', []);
```

Comment:
When `setup` is exposed from `Connection` (see other comment) then just use it here 👍